workers/unity/Assets/Playground/Scripts/Workers/UnityClient.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Generic;
```

Comment:
Nothing else changed in the fine. Is this import needed?

Reply:
Yeah - it was missing before 😬